index.js

```diff
@@ -205,7 +210,7 @@ export const buySubscription = (sku, oldSku, prorationMode) => {
 export const requestSubscription = (sku, oldSku, prorationMode) => Platform.select({
```

Comment:
@solidfox I think there should be a `param` for `andDangerouslyFinishTransactionAutomatically` here too.

Reply:
Agreed, like above.

Reply:
This might be useful: https://developer.apple.com/videos/play/wwdc2018/704/?time=1298